lua/telescope/mappings.lua

```diff
@@ -201,7 +201,11 @@ mappings.execute_keymap = function(prompt_bufnr, keymap_identifier)
     )
   )
 
-  key_func(prompt_bufnr)
+  if type(key_func) == "table" then
+    key_func.fn(prompt_bufnr)
+  else
+    key_func(prompt_bufnr)
+  end
```

Comment:
After you do the fix about `__call` I don't think you'll need this anymore :smile:

Reply:
Yeah you are right, works like a charm.